{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/niohttp2\/niohttp2errors\/pseudoheaderafterregularheader\/location"]}],"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/PseudoHeaderAfterRegularHeader"]]},"metadata":{"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"modules":[{"name":"NIOHTTP2"}],"externalID":"s:8NIOHTTP20A6ErrorsO024PseudoHeaderAfterRegularD0V8locationSSvp","title":"location","roleHeading":"Instance Property","symbolKind":"property"},"primaryContentSections":[{"declarations":[{"platforms":["Linux"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","abstract":[{"type":"text","text":"The file and line where the error was created."}],"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/PseudoHeaderAfterRegularHeader\/location"},"schemaVersion":{"minor":3,"major":0,"patch":0},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"title":"NIOHTTP2","type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","abstract":[],"role":"collection","kind":"symbol","url":"\/documentation\/niohttp2"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/PseudoHeaderAfterRegularHeader":{"abstract":[{"type":"text","text":"A header block contained a pseudo-header after a regular header."}],"title":"NIOHTTP2Errors.PseudoHeaderAfterRegularHeader","role":"symbol","type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/PseudoHeaderAfterRegularHeader","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"PseudoHeaderAfterRegularHeader"}],"url":"\/documentation\/niohttp2\/niohttp2errors\/pseudoheaderafterregularheader","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"PseudoHeaderAfterRegularHeader"}]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors":{"abstract":[{"type":"text","text":"Errors that "},{"type":"reference","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","isActive":true},{"type":"text","text":" raises when handling HTTP\/2 connections."}],"title":"NIOHTTP2Errors","kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NIOHTTP2Errors"}],"type":"topic","url":"\/documentation\/niohttp2\/niohttp2errors","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"NIOHTTP2Errors"}]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/PseudoHeaderAfterRegularHeader/location":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/PseudoHeaderAfterRegularHeader\/location","url":"\/documentation\/niohttp2\/niohttp2errors\/pseudoheaderafterregularheader\/location","kind":"symbol","role":"symbol","type":"topic","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"location"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"}],"abstract":[{"text":"The file and line where the error was created.","type":"text"}],"title":"location"}}}